### DISCUSSION TITLE
pls port forge 1 16 5 pls

### DISCUSSION THREAD

**Discussion by alserr21** [Ladysnake/Requiem#508] [Q&A] [answered]
pls port on 1 16 5 forge

**Comment by grayWolfGameingNo2** [comment]
I second this, although I'm not so shure it will happen. there is a number of things that prevents a port to forge, one that I know of is the incompatibility with Optifine. I don't know much about the creation of mods(just starting... creating a moddpack with 116+ mods), but there might be a work around in enough time, effort, manpower/collaboration.

**Comment by Redfan2** [comment, marked as answer]
Ladysnake members have stated multiple times that they are not comfortable with ports of their Mods to Forge by any party and don‘t plan to do it themselves
